### PR TITLE
[jsk_apc2016_common] add RBOSegmentationInBin

### PR DESCRIPTION
**Pull request by yuyu2172**

### CODE REVIEW

**Comment by yuyu2172** [comment]
@knorth55 
I mistakenly closed this PR yesterday.
Please check :sweat:

**Comment by knorth55** [comment]
PRのタイトルは"Add RBOSegmentationInBin"とかでいいんじゃないですか？
あとで検索しやすいように

File: jsk_apc2016_common/python/jsk_apc2016_common/segmentation_in_bin/rbo_segmentation_in_bin.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import pickle
+import cv2
+
+rospack = rospkg.RosPack()
+pack_path = rospack.get_path('jsk_apc2016_common')
+
+
+class RBOSegmentationInBin(object):
+    """
+    """
```
Comment (knorth55) [inline jsk_apc2016_common/python/jsk_apc2016_common/segmentation_in_bin/rbo_segmentation_in_bin.py]: no description?

Comment (yuyu2172) [inline jsk_apc2016_common/python/jsk_apc2016_common/segmentation_in_bin/rbo_segmentation_in_bin.py]: A little bit is added.

**Comment by knorth55** [comment]
+1

File: jsk_apc2016_common/python/jsk_apc2016_common/segmentation_in_bin/rbo_segmentation_in_bin.py (hunk 1 above)
Comment (knorth55) [inline jsk_apc2016_common/python/jsk_apc2016_common/segmentation_in_bin/rbo_segmentation_in_bin.py]: better

**Comment by yuyu2172** [comment]
@wkentaro
Please merge

**Pull request merged**